net/nfdump/Makefile

```diff
@@ -40,7 +40,9 @@ define Package/nfdump/conffiles
 endef
 
 TARGET_CFLAGS += -flto
+
 TARGET_LDFLAGS += -Wl,--as-needed $(if $(CONFIG_USE_MUSL),-lfts)
+TARGET_LDFLAGS += -L$(STAGING_DIR)/usr/lib
```

Comment:
Isn't this added by default?

Reply:
as far as I can tell, no. I only see the toolchain library directory. It would not build on my system until I did this.

Reply:
I'm going to take a deeper look really quick because I'm not 100% certain what makes this necessary now...

Reply:
nfdump uses macro

`AC_PROG_LEX(yywrap)`

and autoconf has these changes (second one partially reverts the first)

https://git.savannah.nongnu.org/cgit/autoconf.git/commit/?id=29ede6b96feee29c0c477d1659081bbdb82cd8b3
https://git.savannah.nongnu.org/cgit/autoconf.git/commit/?id=46f384f850b455cdb366bafbcf4e992606b3f2c0

Reply:
uci has the same thing

```
$ git grep TARGET_LDFLAGS | grep usr
package/libs/openssl/Makefile:  [ -n "$(TARGET_LDFLAGS)" ] && $(SED) 's#$(TARGET_LDFLAGS)##g' $(1)/usr/lib/pkgconfig/{openssl,libcrypto,libssl}.pc || true
package/system/uci/Makefile:TARGET_LDFLAGS += -L$(STAGING_DIR)/usr/lib
rules.mk:    TARGET_LDFLAGS+= -L$(TOOLCHAIN_DIR)/usr/lib -L$(TOOLCHAIN_DIR)/lib
```

Reply:
that...sounds like a bug.

Reply:
do you think `-L$(STAGING_DIR)/usr/lib` should be in rules.mk? I kind of had that thought but didnt want to bother with it

Reply:
I thought it already was. Weird.

Reply:
Gonna ask @jow- about this.